_includes/index/splash-in-numbers.html

```diff
@@ -26,7 +26,7 @@
                 $(this).prop('Counter', 0).animate({
                     Counter: $(this).text()
                 }, {
-                    duration: 1000,
```

Comment:
This increases the duration of the animation with the numbers increasing from 1s to 2s.